### PR TITLE
Fix session sticky issue

### PR DESCRIPTION
**Pull request by tusharshahrs**
Screen Shot of haproxy.cfg with sticky session turned ON:
tusharshah_dev-digital-deals-haproxy-test-nil-01____ _-ssh_-l_tusharshah_ec2-34-225-138-136_compute-1_amazonaws_com_ _228x39

dev-digital-deals-haproxy-test-nil-01_-_server_inputs

Screen Shot of haproxy.cfg with sticky session turned OFF:
tusharshah_dev-digital-deals-haproxy-test-nil-02____ _-ssh_-l_tusharshah_ec2-34-225-138-136_compute-1_amazonaws_com_ _241x37_and_tusharshah_dev-digital-deals-haproxy-test-nil-01____ _-ssh_-l_tusharshah_ec2-34-225-138-136_compute-1_amazonaws

dev-digital-deals-haproxy-test-nil-02_-_server_inputs

### CODE REVIEW

File: haproxy/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@ this is the changelog for Server Templates
 
 v2.0.2
 -----
+- fixing sticky session false to nil 
+
+v2.0.2
```
Comment (cdwilhelm) [inline haproxy/CHANGELOG.md]: This change should be above the v2.0.2 version and this version should be v2.0.3

**Review by cdwilhelm — CHANGES_REQUESTED** [review]

**Review by rshade — DISMISSED** [review]
this should just be updated to use boolean values

File: haproxy/CHANGELOG.md (hunk 1 above)
Comment (tusharshahrs) [reply haproxy/CHANGELOG.md]: Updated

**Comment by tusharshahrs** [comment]
Here is haproxy with sticky session set to true
haproxy-test-nil-01

Here is the ServerTemplate setting set to true:
haproxy-test-nil-01-servertemplate

Here is haproxy with sticky session set to false
haproxy-test-nil-02



Here is the ServerTemplate setting set to false:
dev-digital-deals-haproxy-test-nil-02_-_server_inputs

**Comment by tusharshahrs** [comment]
Updated for boolean

**Review by cdwilhelm — APPROVED** [review]
good work!

**Pull request merged**